cfgov/v1/wagtail_hooks.py

```diff
@@ -9,6 +9,7 @@
 @hooks.register('after_create_page')
 @hooks.register('after_edit_page')
 def share_the_page(request, page):
```

Comment:
Also, when I put in `print type(page)` here it prints the type of page it is, not a Page object. Where were you seeing this as a Page object?